cfme/tests/cloud/test_provisioning.py

```diff
@@ -207,6 +207,112 @@ def _finished():
         num_sec=600, delay=5, message="VM {} becomes visible".format(vm_name))
 
 
+def test_ec2_manual_placement_using_rest(
+        request, setup_provider, provider, vm_name, rest_api, provisioning):
+    """ Tests provisioning ec2 instance with manual placement using the REST API.
+
+    Metadata:
+        test_flag: provision
+    """
+    image_guid = rest_api.collections.templates.get(name=provisioning['image']['name']).guid
+    provider_rest = rest_api.collections.providers.get(name=provider.name)
+    security_group_name = provisioning['security_group'].split(':')[0].strip()
+    instance_type = provisioning['instance_type'].split(':')[0].strip()
+
+    flavors = rest_api.collections.flavors.find_by(name=instance_type)
+    assert len(flavors) > 0
+    flavor = None
+    for flavor in flavors:
+        if flavor.ems_id == provider_rest.id:
+            break
+    else:
+        pytest.fail("Cannot find flavour.")
+
+    provider_data = rest_api.get(provider_rest._href +
+        '?attributes=cloud_networks,cloud_subnets,security_groups')
```

Comment:
I could have used ``rest_api.get_entity(rest_api.collections.providers, provider_rest.id, attributes="...")``, but I wanted to get just unprocessed dictionary. I had a mysterious issue with missing keys and wanted to be absolutely sure it's not stripped during result processing in the api client.